software/control_sw/scripts/rest_serve_remoteobjects.py

```diff
@@ -56,6 +56,12 @@ def get(self):
     parser.add_argument('--program', '-p', action='store_true',
         help='Program the PCI devices with fpgfile.'
     )
+    parser.add_argument('--redis-host', type=str, default='cosmic-head',
```

Comment:
Can we standardize on `redishost` as the default redishost (which is much easier to change in DNS if the server moves)